src/API/Reports/Customers/DataStore.php

```diff
@@ -626,6 +625,34 @@ public static function get_customer_id_by_user_id( $user_id ) {
 		return $customer_id ? (int) $customer_id : false;
 	}
 
+	/**
+	 * Retrieve the last order made by a customer.
+	 *
+	 * @param int $customer_id Customer ID.
+	 * @return object Order|false.
+	 */
+	public static function get_last_order( $customer_id ) {
+		global $wpdb;
+		$orders_table                = $wpdb->prefix . 'wc_order_stats';
+		$excluded_statuses_condition = "AND status IN ( '" . implode( "','", array_map( 'esc_sql', array_keys( wc_get_order_statuses() ) ) ) . "' )";
```

Comment:
This variable name seems opposite of how it's being used. Do we need to filter the status here anyhow?

Reply:
Yeah I don't think the status filter is necessary given this table only contains orders anyway.